sre-recipes/recipe.py

```diff
@@ -66,6 +66,16 @@ def _run_command(command):
         )
         output, error = process.communicate()
         return output, error
+    
+    @staticmethod
+    def _run_command_interactive(command):
+        """Runs the given interactive command that waits for user input and returns any output and error"""
+        process = subprocess.Popen(
+            command.split(), shell=True, stderr=subprocess.PIPE
```

Comment:
maybe also add `stdout=subprocess.PIPE`

Reply:
In this case, that the user will be promoted to choose the project they want to delete.

Reply:
After further investigation it might be better to use `run` instead i.e. `process =subprocess.run(command.split())` thoughts?